website/source/docs/guides/connect-production.md

```diff
@@ -184,21 +150,38 @@ With [necessary ACL tokens](#configure-acls) in place, all service registrations
 need to have an appropriate ACL token present.
 
 For on-disk configuration the `token` parameter of the service definition must
-be set.
+be set. 
+
+```json
+{ 
+  "service": { 
+    "name": "cassandra_db", 
+    "port": 9002, 
+    "token: "<your_token_here>"
+    } 
+ }
+```
 
-For registration via the API [the token is passed in the request
-header](/api/index.html#acls) or by using the [Go
+For registration via the API the token is passed in the [request
+header](/api/index.html#authentication), `X-Consul-Token`, or by using the [Go
 client configuration](https://godoc.org/github.com/hashicorp/consul/api#Config).
 
-For examples of proxy service definitions see the [proxy
-documentation](/docs/connect/proxies.html).
-
 To avoid the overhead of a proxy, applications may [natively
 integrate](/docs/connect/native.html) with connect.
 
-### Protect Application Listener
-
-If using any kind of proxy for connect, the application must ensure no untrusted
+~> Protect Application Listener: If using any kind of proxy for connect, the application must ensure no untrusted
 connections can be made to it's unprotected listening port. This is typically
 done by binding to `localhost` and only allowing loopback traffic, but may also
 be achieved using firewall rules or network namespacing.
```

Comment:
Does all of this render as a note given the line brea? I only looked at the diff so if you checked with middleman rendered version locally all good.

Reply:
I double checked and its all in the callout!